sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/binaryfile/BinaryFileFormatSuite.scala

```diff
@@ -286,4 +292,45 @@ class BinaryFileFormatSuite extends QueryTest with SharedSQLContext with SQLTest
       EqualTo(MODIFICATION_TIME, file1Status.getModificationTime)
     ), true)
   }
+
+
+  test("genPrunedRow") {
```

Comment:
Can we just test `buildReader` on one file?

Reply:
Do we need (and how to) test when pruned, the file is actually not read ?